### PR TITLE
Mobile 4470

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: src/core/components/combobox/combobox.ts [hunk 1]
```diff
@@ -50,14 +49,26 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
...
     @Input() label = Translate.instant('core.show'); // Aria label.
     @Input() disabled = false;
     @Input() selection = '';
     @Output() onChange = new EventEmitter<unknown>(); // Will emit an event the value changed.
 
     // Additional options when interface modal is selected.
     @Input() icon?: string; // Icon for modal interface.
+
+    /**
+     * Interface to use, popover or modal.
```
Comment (NoelDeMartin) [inline src/core/components/combobox/combobox.ts:68]: I think this description is not right, given that it is the same as the `interface` input.

File: src/core/components/combobox/combobox.ts [hunk 2]
```diff
@@ -50,14 +49,26 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 })
 export class CoreComboboxComponent implements ControlValueAccessor {
 
+    /**
+     * Interface to use, popover or modal.
+     *
+     * @deprecated since 4.4. In a near future, modals won't be supported.
```
Comment (NoelDeMartin) [inline src/core/components/combobox/combobox.ts:55]: Should we add this to upgrade.txt? Plugins can use this component, right?

File: src/theme/components/ion-item.scss [hunk 3]
```diff
@@ -39,16 +39,12 @@ ion-item.item {
         }
     }
 
+
```
Comment (NoelDeMartin) [inline src/theme/components/ion-item.scss:42]: It seems like this newline was added by mistake.

File: src/core/components/combobox/combobox.ts (hunk 1 above)
Comment (crazyserver) [reply src/core/components/combobox/combobox.ts:68]: Sorry, I've double checked the usage of the component in workplace and we still use the modal option there so I'm not going to deprecate this yet.

**Review by NoelDeMartin — APPROVED** [review]

**Pull request merged**